src/config.rs

```diff
@@ -36,21 +36,25 @@ fn init_env_cfg() -> Result<EnvCfg> {
 
 pub static ENV_CFG: Lazy<EnvCfg> = Lazy::new(|| init_env_cfg().expect("Initialize env config"));
 
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, Clone)]
 pub struct Volume {
     pub path: PathBuf, // Path to mounted volume
 }
 
 #[derive(Deserialize)]
 struct SysCfgFile {
+    http_port: Option<u16>,
     private_key: Option<SecretKey>,
+    drive_redundancy: Option<usize>,
     volumes: Option<Vec<Volume>>,
     capacity: Option<u64>,
 }
 
 #[derive(Serialize)]
 pub struct SysCfg {
+    pub http_port: u16,
     pub private_key: SecretKey,
+    pub drive_redundancy: usize,
```

Comment:
dirve_redundancy is usize?
Why not u8 as a type?

Reply:
It could be, but it's not necessary, it's only used at runtime. That change would be more useful if it were serialized to a fixed amount of bytes on-disk, but this is just serialized to TOML.